website/content/en/v1.0/upgrading/v1-migration.md

```diff
@@ -299,6 +299,18 @@ You should still review the upgrade procedure; the sequence of operations remain
 Karpenter has deprecated and moved a number of Helm values as part of the v1 release. Ensure that you upgrade to the newer version of these helm values during your migration to v1. You can find detail for all the settings that were moved in the [v1 Upgrade Reference]({{<ref "#helm-values" >}}).
     {{% /alert %}}
 
+    {{% alert title="Note" color="primary" %}}
+<!-- Note: don't indent this line to match the indenting of the alert box. Hugo will create a code block. -->
+Karpenter 0.37.x has a number of mutating webhooks and validating webhooks that are not present in 1.0.x; if you are
```

Comment:
We want to inform users that this is present for v0.32.x - v0.37.x

Reply:
Updated.